go.mod

```diff
@@ -9,35 +9,41 @@ replace (
 
 require (
 	github.com/Maldris/mathparse v0.0.0-20170508133428-f0d009a7a773
-	github.com/ethereum/go-ethereum v1.12.0
+	github.com/ethereum/go-ethereum v1.13.8
 	github.com/go-echarts/go-echarts/v2 v2.2.6
 	github.com/google/uuid v1.3.1
 	github.com/jedib0t/go-pretty/v6 v6.4.7
 	github.com/pkg/errors v0.9.1
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240118012339-4864e2306bb1
-	github.com/smartcontractkit/libocr v0.0.0-20230925165524-ffa38fe11ef8
+	github.com/smartcontractkit/libocr v0.0.0-20240229181116-bfb2432a7a66
```

Comment:
there's a newer commit `5a9177221bec36ac3bf76a161c81a3389d5b17b6`  in libocr master. Do we want to target that?